internal/disk/disk.go

```diff
@@ -0,0 +1,64 @@
+package disk
+
+import (
+	"gobar/internal/blockutils"
+	"gobar/internal/clickutils"
+	"gobar/internal/log"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// GetDisk returns a block containing the remaining space for the home partition
+func GetDisk(timeout time.Duration, blockCh chan<- *blockutils.Block) {
+	const hddRune = '\uf51f'
+	var diskSpace string
+	diskBlock := blockutils.Block{
+		Name:        blockutils.DiskName,
+		Border:      blockutils.Red,
+		BorderLeft:  0,
+		BorderRight: 0,
+		BorderTop:   0,
+		Urgent:      false,
+		FullText:    "",
+	}
+
+	for {
+		cmd := exec.Command("df", "-h")
+		outLines, err := blockutils.RunCmdStdout(cmd)
+		if err != nil {
+			log.FileLog(err)
+		}
+
+		if outLines != nil {
+			for _, line := range outLines {
+				if strings.HasSuffix(line, "/home") {
```

Comment:
there are plans afoot with systemd to move this - might want to add flexibility eventually

Reply:
To remove mount points from df output?

Reply:
maybe that too - systemd is built from evil

systemd-home will mount your home directory - likely at some strange systemd system directory - so /home will eventually become a non-thing